### ISSUE TITLE
[studio-ui] Make it easy to edit viewed files

### ISSUE THREAD

**Issue by sumerjabri** [craftercms/craftercms#5266]
#### Is your feature request related to a problem? Please describe.
It would be great if you can edit a file after viewing it without having to go back to the sidebar and choose edit.

#### Describe the solution you'd like
Clicking on a template, script or other editable artifacts would show the view mode, and then present a small button that lets you edit the artifact.

#### Describe alternatives you've considered
{{A clear and concise description of any alternative solutions or features you've considered.}}

**Comment by sumerjabri** [comment]
@aachavarria please make sure the edit panel is in focus when the user clicks on Edit.

Also, did you check permissions? The checkbox above is not set.

**Comment by aachavarria** [comment]
Yes im checking permissions, but for Roy recommendation the edit button is not visible if you dont have the edit permission

**Comment by sumerjabri** [comment]
Will validate focus issues in the ticket #5287

**Comment by sumerjabri** [comment]
@rart when opening in view-only, the UI isn't clear that this is a view-only mode. Ping me to discuss.

**Comment by sumerjabri** [comment]
@rart please make it darker in Edit mode so it would work for different monitors.

**Comment by rart** [comment]
At this point, this ticket and https://github.com/craftercms/craftercms/issues/5312 are duplicates. Close one, @sumerjabri?
